### PR TITLE
move cluster to the correct subnet

### PR DESCRIPTION
**Pull request by StuartHarris**
Moves the `non-prod` cluster into the correct subnet. This also entails moving to `uksouth` (from `westeurope`)

### CODE REVIEW

File: .github/workflows/doc-index-updater.yaml [hunk 1]
```diff
@@ -19,23 +19,23 @@ jobs:
...
-        working-directory: ./products/medicines/doc-index-updater
-        run: |
-          TAG="$(git rev-parse --short=7 ${{ github.sha }})"
-          make ci-branch tag=$TAG
+      # - name: Build and test Docker image (BRANCH only)
+      #   if: github.ref != 'refs/heads/master'
+      #   working-directory: ./products/medicines/doc-index-updater
+      #   run: |
+      #     TAG="$(git rev-parse --short=7 ${{ github.sha }})"
+      #     make ci-branch tag=$TAG
```
Comment (pataruco) [inline .github/workflows/doc-index-updater.yaml]: I think you left this comment to speed up your feedback loop

**Comment by StuartHarris** [comment]
Don't merge this yet. I removed the `development` and `prod` environments because they were getting out of date and we really only have a `non-prod` cluster. However I forgot about the CDN configuration in #233 which we do need. I think we have to bring that PR in first, and fix the `prod` environment so that it just has CDN (and other relevant) stuff.

**Comment by StuartHarris** [comment]
I reinstated the prod env (without a cluster) so that we don't lose the current production configuration and we will be able to merge #233 to master. So I think this can be reviewed now

File: infrastructure/environments/prod/main.tf [hunk 2]
```diff
@@ -44,17 +40,6 @@ module cpd {
 
   environment         = var.ENVIRONMENT
   location            = var.REGION
+  namespace           = local.namespace
   resource_group_name = azurerm_resource_group.products.name
 }
-
-# Service Bus
-module service_bus {
```
Comment (sgrowe) [inline infrastructure/environments/prod/main.tf]: We don't want a service bus or cluster in prod?

**Comment by StuartHarris** [comment]
@sgrowe We do indeed need service bus in production, but I wanted to remove all the resources that we haven't built yet in production, as non-prod was changing and keeping them both in sync is currently unneccessary. I think it will be easier to update prod config later when it's settled down.

File: infrastructure/environments/prod/variables.tf [hunk 3]
```diff
@@ -8,15 +8,7 @@ variable "RESOURCE_GROUP_PRODUCTS" {
...
-}
-
-variable "CLIENT_SECRET" {
-  description = "Service Principal Client Secret"
-}
-
 variable "ENVIRONMENT" {
   description = "Environment name"
-  default     = "prod"
+  default     = "non-prod"
```
Comment (sgrowe) [inline infrastructure/environments/prod/variables.tf]: How come you've renamed this to non-prod? 🤔 

If the clusters are now named after `var.environment` won't that cause a clash if we try to spin them both up?

Comment (StuartHarris) [reply infrastructure/environments/prod/variables.tf]: Thanks, fixed that now

**Pull request merged**